features/option-list.feature

```diff
@@ -38,13 +38,24 @@ Feature: List WordPress options
     When I run `wp option add sample_test_field_one sample_test_field_value_one`
     And I run `wp option add sample_test_field_two sample_test_field_value_two`
     And I run `wp option list --search="sample_test_field_*" --format=csv`
+    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_name --order=asc`
+    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_value --order=asc`
```

Comment:
The ordering is different at the end. This produces two different lines of output, that are tested below.